sdk/storage/azblob/blockblob/mmf_windows.go

```diff
@@ -0,0 +1,52 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package blockblob
+
+import (
+	"fmt"
+	"os"
+	"reflect"
+	"syscall"
+	"unsafe"
+)
+
+// mmb is a memory mapped buffer
+type mmb []byte
+
+// newMMB creates a new memory mapped buffer with the specified size
+func newMMB(size int64) (mmb, error) {
+	const InvalidHandleValue = ^uintptr(0)
```

Comment:
This should be -1; not 0

Reply:
The `^` inverts the bits (see also [here](https://github.com/golang/go/blob/master/src/runtime/netpoll_windows.go#L14)).

Reply:
I missed the ^. How about a comment // -1